bin/installInfo.js

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env node
+'use strict';
+console.log('##########################################');
+console.log('# Please add the following typings manually:');
+console.log('# typings install dt~es6-collections dt~es6-promise dt~backbone dt~cordova/plugins/device dt~phonegap-plugin-push dt~websql dt~backbone-global dt~q dt~jquery dt~underscore dt~node --save --global');
```

Comment:
The relution-sdk does not depend on es6-collections and es6-promise, so users of the library don't need to install these. (Your project probably uses these by itself, but this is not an SDK requirement, so just remove these two from the message.)